server/src/main/java/org/opensearch/index/seqno/LocalCheckpointTracker.java

```diff
@@ -134,6 +134,29 @@ public synchronized void markSeqNoAsPersisted(final long seqNo) {
         markSeqNo(seqNo, persistedCheckpoint, persistedSeqNo);
     }
 
+    /**
+     * Updates the processed checkpoint with the provided sequence number if segment replication is enabled
+     *
+     * @param seqNo the sequence number to mark as processed
+     */
+    public synchronized void segrepMarkSeqNoAsProcessed(final long seqNo) {
```

Comment:
A suggestion here, can we name this after what it's doing (fast forwarding to seqNo) vs the replication strategy that invokes it?